tmk_core/protocol/vusb/vusb.c

```diff
@@ -597,7 +592,6 @@ const PROGMEM usbConfigurationDescriptor_t usbConfigurationDescriptor = {
         .bDescriptorType     = USBDESCR_HID_REPORT,
         .wDescriptorLength   = sizeof(mouse_extra_hid_report)
     },
-#        if USB_CFG_HAVE_INTRIN_ENDPOINT3
```

Comment:
I cant mark it, but
`        .bNumEndpoints       = USB_CFG_HAVE_INTRIN_ENDPOINT3,`
above should become
`        .bNumEndpoints       = 1,`